datajunction-clients/python/datajunction/builder.py

```diff
@@ -101,21 +101,6 @@ def restore_node(self, node_name: str) -> None:
     #
     # Nodes: SOURCE
     #
-    def source(self, node_name: str) -> "Source":
```

Comment:
@agorajek fyi, I moved these to the regular non-builder client because I figured they're meant to be read-only client objects. This way users can do something like:
```
repair_order_dim = dj.dimension("default.repair_order")
repair_order_dim.linked_nodes()
> [ ... ]
```

Reply:
Sounds good!